gitter/gitter.go

```diff
@@ -1,40 +1,11 @@
 package gitter
 
 import (
-	"context"
-	"fmt"
+	"log"
 
-	"github.com/machinebox/graphql"
+	"git_applet/status"
 )
 
-func ApprovePullRequest(url string, token string, ctx context.Context, id string, body string) error { // TODO: fix the graphql injection xD
-
-	req := graphql.NewRequest(fmt.Sprintf(`mutation {
-		addPullRequestReview(input: {
-		  pullRequestId: "%s",
-		  event: APPROVE,
-		  body: "%s"
-		}) {
-		  pullRequestReview {
-			id
-			url
-		  }
-		}
-	  }`, id, body))
```

Comment:
added to [#45](https://github.com/the-chaos-collective/appletini/issues/45#issue-2709576026)